utils/commands.ts

```diff
@@ -83,6 +84,35 @@ export async function resetWP(): Promise<void> {
 
 }
 
+/**
+ * Check if WP i
+ *
+ * @function
+ * @name resetWP
+ * @async
+ * @returns {Promise<void>} - A Promise that resolves when check is complete.
+ */
+export async function checkWPStatus(): Promise<void> {
+    const status: boolean =  await wp('--info');
+
+    console.log(chalk.blue.bold('WP CLI Basic Setup Status'));
+    console.log(chalk.blue('=====================\n'));
+
+    let statusColor = chalk.green,
+        statusSymbol =  '✔️',
+        message = 'WP CLI is running';
+
+    if(!status) {
+        statusColor =  chalk.red;
+        statusSymbol =  '❌';
+        message = 'WP CLI is not running, please check your config'
+    }
+
+    console.log(`'WP CLI health check': ${statusColor.bold(message.toUpperCase())} ${statusSymbol}`);
+    console.log(statusColor(message));
+    console.log(chalk.blue('-------------------------'));
+}
```

Comment:
The second point is added, using the existing log in and log out feature, I guess that will change a bit since we're moving away from the feature approach